selfdrive/controls/controlsd.py

```diff
@@ -521,7 +541,12 @@ def publish_logs(self, CS, start_time, actuators, v_acc, a_acc, lac_log):
     alerts = self.events.create_alerts(self.current_alert_types, [self.CP, self.sm, self.is_metric])
     self.AM.add_many(self.sm.frame, alerts, self.enabled)
     self.AM.process_alerts(self.sm.frame, clear_event)
-    CC.hudControl.visualAlert = self.AM.visual_alert
+
+    if self.debug_mode and len(self.sm['testJoystick'].buttons) and \
```

Comment:
Do we really want this steer required using a button? Doesn't seem that useful and adds a bunch of code.